src/Features/CSharp/Portable/FullyQualify/CSharpFullyQualifyService.cs

```diff
@@ -45,8 +44,7 @@ protected override async Task<SyntaxNode> ReplaceNodeAsync(SimpleNameSyntax simp
             var newName = simpleName.WithLeadingTrivia(SyntaxTriviaList.Empty);
 
             var qualifiedName = SyntaxFactory.QualifiedName(SyntaxFactory.ParseName(containerName), newName)
-                .WithLeadingTrivia(leadingTrivia)
-                .WithAdditionalAnnotations(Formatter.Annotation);
+                .WithLeadingTrivia(leadingTrivia);
```

Comment:
Shouldn't we just copy all the trivia over from the old node?  I imagine we're can construct a case where the trailing trivia is relevant, but not preserved otherwise.

Reply:
➡️ Trailing trivia is preserved already because the last token doesn't change during full qualification.

Reply:
Ah ok. That makes sense :-)